test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -470,6 +470,53 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
             $LogPath | Should -FileContentMatch 'True'
         }
     }
+
+    Context "Tests around Windows PowerShell Compatibility module deny list" {
+        BeforeAll {
+            $pwsh = "$PSHOME/pwsh"
+            Add-ModulePath $basePath
+        }
+
+        AfterAll {
+            Restore-ModulePath
+        }
+
+        It "Successfully imports incompatible module when DenyList is not specified in powershell.config.json" {
+            $ConfigPath = Join-Path $TestDrive 'powershell.config.json'
```

Comment:
Since it appears this is the same everywhere, put this in BeforeAll

Reply:
Good point; moved to BeforeAll.